modules/preview-react/table/lib/TableHead.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import {GridProps, Grid} from '@workday/canvas-kit-react/layout';
+import {createComponent, styled, StyledType} from '@workday/canvas-kit-react/common';
+
+const StyledHead = styled(Grid.as('thead'))<StyledType>({
+  'th:last-of-type': {
+    borderRight: 'none',
```

Comment:
do you need borderRight none here?

Reply:
So like above, this would be removing the last cell's `border-right`. This will be updated for left and right just in case.